### PR TITLE
[9.0.0] Remove deprecated modules and features

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Remove deprecated modules that are scheduled for removal in community.general 9.0.0.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
rax* modules
stackdriver
webfaction_* modules

### CODE REVIEW

**Comment by felixfontein** [comment]
@russoz django_manage mentioned in its notes:

> B(ATTENTION - DEPRECATION): Support for Django releases older than 4.1 will be removed in
> community.general version 9.0.0 (estimated to be released in May 2024).
> Please notice that Django 4.1 requires Python 3.8 or greater.

Is there anything to implement here than what I did in ceb947ea182b68c8bc1a6271967a1f3174bb95c2?

File: plugins/modules/redfish_command.py [hunk 1]
```diff
@@ -109,9 +109,10 @@
   timeout:
     description:
       - Timeout in seconds for HTTP requests to OOB controller.
-      - The default value for this param is C(10) but that is being deprecated
-        and it will be replaced with C(60) in community.general 9.0.0.
+      - The default value for this parameter changed from C(10) to C(60)
```
Comment (russoz) [inline plugins/modules/redfish_command.py:112]:
```suggestion
      - The default value for this parameter changed from V(10) to V(60)
```

File: plugins/modules/redfish_config.py [hunk 2]
```diff
@@ -64,9 +64,10 @@
   timeout:
     description:
       - Timeout in seconds for HTTP requests to OOB controller.
-      - The default value for this param is C(10) but that is being deprecated
-        and it will be replaced with C(60) in community.general 9.0.0.
+      - The default value for this parameter changed from C(10) to C(60)
```
Comment (russoz) [inline plugins/modules/redfish_config.py:67]:
```suggestion
      - The default value for this parameter changed from V(10) to V(60)
```

File: plugins/modules/redfish_info.py [hunk 3]
```diff
@@ -63,9 +63,10 @@
   timeout:
     description:
       - Timeout in seconds for HTTP requests to OOB controller.
-      - The default value for this param is C(10) but that is being deprecated
-        and it will be replaced with C(60) in community.general 9.0.0.
+      - The default value for this parameter changed from C(10) to C(60)
```
Comment (russoz) [inline plugins/modules/redfish_info.py:66]:
```suggestion
      - The default value for this parameter changed from V(10) to V(60)
```

**Review by russoz — COMMENTED** [review]
Couple of doc adjustments

**Comment by russoz** [comment]
> @russoz django_manage mentioned in its notes:
> 
> > B(ATTENTION - DEPRECATION): Support for Django releases older than 4.1 will be removed in
> > community.general version 9.0.0 (estimated to be released in May 2024).
> > Please notice that Django 4.1 requires Python 3.8 or greater.
> 
> Is there anything to implement here than what I did in ceb947ea182b68c8bc1a6271967a1f3174bb95c2?

I will be taking a closer look on that next days

**Review by omgjlk — APPROVED** [review]
approving removal of rax modules

**Comment by russoz** [comment]
@felixfontein I have raised a PR against your branch, just documentation adjustments.

https://github.com/felixfontein/community.general/pull/7

**Comment by felixfontein** [comment]
Since the upcoming 8.6.0 release will be the last 8.x.0 feature release before 9.0.0, I'm going to merge this PR after the release happened.

**Pull request merged**